### PR TITLE
Document the PSL Python Library #78

### PR DESCRIPTION
**Pull request by fragm3**
Fixes: #78 
Added documentation for [sciencelab.py](https://github.com/fossasia/pslab-python/blob/development/PSL/sciencelab.py) in markdown format

### CODE REVIEW

**Comment by fragm3** [comment]
@CloudyPadmal Please review

File: PSL/README.md [hunk 1]
```diff
@@ -1,75 +1,334 @@
...
++ Blocking call that fetches oscilloscope traces from a set of specified channels
++ Arguments
+	+ samples: Number of samples to fetch. Maximum 10000/(total specified channels)
+	+ tg: Timegap between samples in microseconds.
+	+ \*args: Channel names
++ Return: Arrays X(timestamps),Y1,Y2 ...
+
+```
+>>> from pylab import *
+>>> I=sciencelab.ScienceLab()
```
Comment (CloudyPadmal) [inline PSL/README.md]: Let's have a consistence styling. Spaces between = sign

**Comment by fragm3** [comment]
@CloudyPadmal made the changes

**Comment by fragm3** [comment]
@CloudyPadmal @mariobehling please merge this

**Pull request merged**